.travis.yml

```diff
@@ -3,9 +3,5 @@ language: ruby
 rvm:
   - 2.1.4
 
-branches:
-  only:
-    - develop
```

Comment:
I think this is to run travis in `develop` branch or branch that part from `develop`. With this setup `master` will not run om Travis.